.ci/azure-pipelines/env.yml

```diff
@@ -0,0 +1,61 @@
+# Docker
+# Build a Docker image 
+# https://docs.microsoft.com/azure/devops/pipelines/languages/docker
+
+trigger:
+  branches:
+    include:
+    - master
+  paths:
+    include:
+    - .dev/docker/env/Dockerfile
+
+pr: none
+
+schedules:
+- cron: "0 0 * * *"
+  displayName: "Daily midnight build"
+  branches:
+    include:
+    - master
+
+resources:
+- repo: self
+
+variables:
+  dockerHub: "PersonalDockerHub"
+  dockerHubID: "PointCloudLibrary"
+
+stages:
+- stage: BuildAndPush
+  displayName: "Build env"
+  jobs:
+  - job: BuildAndPush
+    timeoutInMinutes: 360
+    displayName: "env docker image: "
+    pool:
+      vmImage: 'ubuntu-latest'
+    strategy:
+      matrix:
+        Ubuntu 16.04:
+          UBUNTU_DISTRO: 16.04
+          tag: 16.04
+        Ubuntu 20.04:
+          UBUNTU_DISTRO: 20.04
```

Comment:
Our "latest Ubuntu" image is built with `VTK_VERSION=7` argument to (somewhat) increase coverage. I'd go for 8, of course, but it's not packaged even in 20.04 :(